sklearn/utils/estimator_checks.py

```diff
@@ -959,10 +959,13 @@ def check_dtype_object(name, estimator_orig):
 
 
 def check_complex_data(name, estimator_orig):
+    rng = np.random.RandomState(42)
```

Comment:
is this change related to this PR? (I'm happy to keep it here anyway)

Reply:
Yes otherwise it would fail because now the validation is delegated to the underlying estimator, but after the CV split of `CalibratedClassifierCV`. Now this test was failing because the Stratified CV split was failing because of the unique values in y that prevented to have balanced "classes" in the validation folds.